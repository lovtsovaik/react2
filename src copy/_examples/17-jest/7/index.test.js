const add = (a, b) => a + b;

test('«add» function should match its snapshot', () => {
    expect(add(2, 6)).toMatchSnapshot();
});
