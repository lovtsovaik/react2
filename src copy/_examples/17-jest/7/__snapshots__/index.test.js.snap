// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`«add» function should match its snapshot 1`] = `8`;
